src/app/listArea/listArea.component.scss

```diff
@@ -10,15 +10,17 @@
   justify-content: space-between;
```

Comment:
Make sure to use the same font-family for the add item buttons and the list items for now.